pages/status.tsx

```diff
@@ -1,19 +1,30 @@
+import { FC, MouseEventHandler, useCallback, useMemo, useState } from 'react'
+import { GetStaticProps } from 'next'
+import Router from 'next/router'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { useFormik } from 'formik'
+import * as Yup from 'yup'
+import { CheckStatusRequestBody } from './api/check-status'
+import { useCheckStatus } from '../hooks/api/useCheckStatus'
+import Layout from '../components/Layout'
 import InputField from '../components/InputField'
 import ActionButton from '../components/ActionButton'
-import { FC, MouseEventHandler, useCallback, useMemo, useState } from 'react'
 import ErrorSummary, {
   ErrorSummaryItem,
   getErrorSummaryItem,
 } from '../components/ErrorSummary'
-import Layout from '../components/Layout'
-import { CheckStatusRequestBody } from './api/check-status'
-import { useCheckStatus } from '../hooks/api/useCheckStatus'
-import { useFormik } from 'formik'
-import * as Yup from 'yup'
-import useTranslation from 'next-translate/useTranslation'
 import StatusInfo from '../components/StatusInfo'
-import { GetStaticProps } from 'next'
-import Router from 'next/router'
+
+const PageWrapper: FC<{ children: JSX.Element }> = ({ children }) => {
```

Comment:
This shouldn't be separated out into another component.

Reply:
Why not? Othwise I would need to duplicate the following code at 3 occasions (each conditional return)

```tsx
<Layout>
    <h1 className="mb-4">{t('header')}</h1>
    ( // conditional content here }
</Layout>
```

Reply:
I will revert it as it was before.